@xen-orchestra/web-core/lib/components/ui/checkbox/UiCheckbox.vue

```diff
@@ -0,0 +1,429 @@
+<!-- v3 -->
+<template>
+  <!-- @TODO: Replace it by the VtsLabel component when update -->
+  <label class="form-checkbox" v-bind="wrapperAttrs">
+    <input
+      v-model="value"
+      :class="[classNames, { indeterminate: isIndeterminate }]"
+      :disabled="isDisabled"
+      type="checkbox"
+      class="ui-checkbox"
+      v-bind="attrs"
+    />
+    <span class="fake-checkbox">
+      <VtsIcon :fixed-width="false" :icon class="icon" accent="brand" />
+    </span>
+    <span v-if="slots.default" class="typo p1-regular">
+      <slot />
+    </span>
+  </label>
+  <VtsInfo v-if="slots.info" :accent>
+    <slot name="info" />
+  </VtsInfo>
+</template>
+
+<script lang="ts" setup>
+import VtsIcon from '@core/components/icon/VtsIcon.vue'
+import VtsInfo from '@core/components/info/VtsInfo.vue'
+import { useContext } from '@core/composables/context.composable'
+import { DisabledContext } from '@core/context'
+import { toVariants } from '@core/utils/to-variants.util'
+import { faCheck, faMinus } from '@fortawesome/free-solid-svg-icons'
+import { computed, type HTMLAttributes, useAttrs } from 'vue'
+
+type CheckboxAccent = 'brand' | 'success' | 'warning' | 'danger'
+
+defineOptions({ inheritAttrs: false })
+
+const props = withDefaults(
+  defineProps<{
+    accent: CheckboxAccent
+    disabled?: boolean
+    wrapperAttrs?: HTMLAttributes
+  }>(),
+  { disabled: undefined }
+)
+
+const value = defineModel<unknown>('modelValue')
+
+const slots = defineSlots<{
+  default?(): any
+  info?(): any
+}>()
+
+const isDisabled = useContext(DisabledContext, () => props.disabled)
+
+const classNames = computed(() => [
+  toVariants({
+    accent: props.accent,
+    disabled: isDisabled.value,
+  }),
+])
+const icon = computed(() => {
+  if (value.value === undefined) {
+    return faMinus
+  }
+
+  return faCheck
+})
+
+const attrs = useAttrs()
+
+const isIndeterminate = computed(() => value.value === undefined)
+</script>
+
+<style lang="postcss" scoped>
+.form-checkbox {
+  display: inline-flex;
+  align-items: center;
+  gap: 0.8rem;
+
+  .fake-checkbox {
+    display: inline-flex;
+    align-items: center;
+    justify-content: center;
+    height: 1.6rem;
+    width: 1.6rem;
+    transition:
+      background-color 0.125s ease-in-out,
+      border-color 0.125s ease-in-out;
+    border: 0.2rem solid transparent;
+    border-radius: 0.2rem;
+    box-shadow: var(--shadow-100);
```

Comment:
In the design system, I don't see a shadow applied to the component, I don't think it is necessary here.

Reply:
Yes it's an oversight from copy/paste 😨